### PR TITLE
Remove never-supported HTML*Element APIs

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6854, #6768

### CODE REVIEW

File: api/HTMLIsIndexElement.json [hunk 1]
```diff
@@ -1,148 +0,0 @@
-{
-  "api": {
-    "HTMLIsIndexElement": {
```
Comment (foolip) [inline api/HTMLIsIndexElement.json:3]: I think this should be paired with removal of https://github.com/mdn/browser-compat-data/blob/master/html/elements/isindex.json.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/isindex says "All major browsers have now removed `<isindex>`" but not when.

Per https://github.com/whatwg/html/pull/1095 support was removed from Edge and Chrome back then, and Firefox was planning to remove. That happened here:
https://github.com/mozilla/gecko-dev/commit/217ac88f04911b1c3c956f7b70f431735ab244ae

WebKit support was dropped here:
https://trac.webkit.org/changeset/205858/webkit

So yes, this is far enough in the past that we can drop it from MDN/BCD.

**Review by foolip — COMMENTED** [review]
I haven't finished reviewing yet, submitting half a review.

File: api/HTMLButtonElement.json [hunk 2]
```diff
@@ -529,54 +529,6 @@
           }
         }
       },
-      "menu": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLButtonElement/menu",
```
[the remark this answers is not in the document]
Comment (saschanaz) [reply api/HTMLButtonElement.json:534]: My investigation says https://github.com/whatwg/html/pull/2342 removed it.

File: api/HTMLIsIndexElement.json (hunk 1 above)
Comment (saschanaz) [reply api/HTMLIsIndexElement.json:3]: It's covered in https://github.com/mdn/browser-compat-data/pull/6955 but I can also do it here. Do you prefer to do it here?

File: api/HTMLButtonElement.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (foolip) [reply api/HTMLButtonElement.json:534]: Thanks for digging that up! Sounds like it was only ever shipped in Gecko, and it was removed here:
https://github.com/mozilla/gecko-dev/commit/6ca867236ba72f2f0dd969e0a6ed4055376d5c88

File: api/HTMLIsIndexElement.json (hunk 1 above)
Comment (foolip) [reply api/HTMLIsIndexElement.json:3]: Nah, let's leave it in the other PR, I didn't know it was covered there.

File: api/HTMLIFrameElement.json [hunk 3]
```diff
@@ -1012,55 +1012,6 @@
           }
         }
       },
-      "setNfcFocus": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/setNfcFocus",
```
Comment (foolip) [inline api/HTMLIFrameElement.json:1017]: I have redirected https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/setNfcFocus to https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement.

File: api/HTMLMenuItemElement.json [hunk 4]
```diff
@@ -95,54 +95,6 @@
           }
         }
       },
-      "command": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLMenuItemElement/command",
```
Comment (foolip) [inline api/HTMLMenuItemElement.json:100]: This is 404, no action needed.

File: api/HTMLTableElement.json [hunk 5]
```diff
@@ -805,90 +805,6 @@
           }
         }
       },
-      "sortable": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLTableElement/sortable",
```
Comment (foolip) [inline api/HTMLTableElement.json:810]: This is 404

File: api/HTMLTableElement.json [hunk 6]
```diff
@@ -805,90 +805,6 @@
...
-          "status": {
-            "experimental": true,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
-      "stopSorting": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLTableElement/stopSorting",
```
Comment (foolip) [inline api/HTMLTableElement.json:852]: This is 404.

File: api/HTMLIsIndexElement.json [hunk 7]
```diff
@@ -1,148 +0,0 @@
...
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": true
-          }
-        }
-      },
-      "prompt": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement/prompt",
```
Comment (foolip) [inline api/HTMLIsIndexElement.json:100]: 404

File: api/HTMLIsIndexElement.json [hunk 8]
```diff
@@ -1,148 +0,0 @@
...
-        },
-        "status": {
-          "experimental": false,
-          "standard_track": false,
-          "deprecated": true
-        }
-      },
-      "form": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement/form",
```
Comment (foolip) [inline api/HTMLIsIndexElement.json:52]: 404

File: api/HTMLIsIndexElement.json [hunk 9]
```diff
@@ -1,148 +0,0 @@
-{
-  "api": {
-    "HTMLIsIndexElement": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement",
```
Comment (foolip) [inline api/HTMLIsIndexElement.json:5]: https://developer.mozilla.org/docs/Web/API/HTMLIsIndexElement will need to be removed.

**Review by foolip — APPROVED** [review]
Over to @chrisdavidmills, https://developer.mozilla.org/en-US/docs/Web/API/HTMLIsIndexElement needs to be removed.

**Comment by chrisdavidmills** [comment]
@foolip https://wiki.developer.mozilla.org/en-US/docs/Web/API/HTMLIsIndexElement has been deleted.

**Review by foolip — APPROVED** [review]
Thanks @chrisdavidmills! I'll go ahead and merge this now.

**Pull request merged**